src/Illuminate/Session/SessionManager.php

```diff
@@ -103,7 +105,18 @@ protected function createApcDriver()
 	 */
 	protected function createMemcachedDriver()
 	{
-		return $this->createCacheBased('memcached');
+		$store = $this->app['config']['session.memcached_store'];
+
+		// Verify store uses the memcached driver

```

Comment:
Please use lower case here. It's not a sentence because it doesn't end with a `.`.